File: clj/fib.clj
(defn fib [n]
  (loop [a 1, b 1, vx [a b]]
    (if (= n (count vx))
	    vx
      (let [ab (+ a b)]
        (recur ab a (conj vx ab))))))

(def nbrs [4 7 14])

(doseq [r (map fib nbrs)] println)
(doseq [n nbrs]
  (println (fib n)))

(run! println (map fib [ 5 10 15]))

File: ktvm/fib.py
import time

def fib(n):
    a, b = 1, 0
    while n > 0:
        c = a
        a = a + b
        b = c
        n -= 1
    return b

start = time.time()

count = 100
while (count > 0):
    x = fib(40)
    count -= 1;

stop = time.time()

print(x, 1000 * 1000 * (stop - start))
